tree/ntuple/v7/inc/ROOT/RPageStorageFile.hxx

```diff
@@ -114,17 +116,35 @@ public:
    static constexpr std::size_t kMaxPageSize = 1024 * 1024;
 
 private:
+   /// Wraps the I/O counters
    RNTupleMetrics fMetrics;
+   RNTupleAtomicCounter *fCtrNReadV = nullptr;
+   RNTupleAtomicCounter *fCtrNRead = nullptr;
+   RNTupleAtomicCounter *fCtrSzReadPayload = nullptr;
+   RNTupleAtomicCounter *fCtrSzReadOverhead = nullptr;
+   RNTuplePlainCounter *fCtrSzUnzip = nullptr;
+   RNTupleAtomicCounter *fCtrNClusterLoaded = nullptr;
+   RNTuplePlainCounter *fCtrNPageLoaded = nullptr;
+   RNTuplePlainCounter *fCtrNPagePopulated = nullptr;
+   RNTupleAtomicCounter *fCtrTimeWallRead = nullptr;
+   RNTuplePlainCounter *fCtrTimeWallUnzip = nullptr;
+   RNTupleTickCounter<RNTupleAtomicCounter> *fCtrTimeCpuRead = nullptr;
+   RNTupleTickCounter<RNTuplePlainCounter> *fCtrTimeCpuUnzip = nullptr;
```

Comment:
Should those me moved and grouped into a dedicated class? If so, will generally all of them be `!= nullptr` or none, or might only some be `nullptr`? If the former, consider writing:
```
struct Counters {
   RNTupleAtomicCounter &fNReadV;
...
};
Counters *fCounters;
```
For the latter, maybe
```
struct Counters {
   RNTupleAtomicCounter *fNReadV;
...
};
Counters fCounters;
```